src/zenml/integrations/label_studio/__init__.py

```diff
@@ -27,8 +27,8 @@ class LabelStudioIntegration(Integration):
 
     NAME = LABEL_STUDIO
     REQUIREMENTS = [
-        "label-studio>=1.6.0,<=1.7.3",
-        "label-studio-sdk>=0.0.17,<=0.0.24",
+        "label-studio>=1.6.0,<=1.9.1",
```

Comment:
Are you sure this still works for older versions? In particular, does `task["storage_filename"]` below exist in all versions?

Reply:
https://github.com/HumanSignal/label-studio/blob/b0df5dd19a5400e8d26fb8269e2add87799af055/label_studio/tasks/models.py#L155 yes the `storage_filename` property exists in >=0.6.0 versions.